src/translations/en.json

```
@@ -2372,6 +2372,15 @@
             "show_encryption_key": {
               "title": "Encryption key",
               "description": "Make sure you save the encryption key in a secure place so you always have access to your backups."
+            },
+            "download": {
+              "decryption_unsupported_title": "Decryption unsupported",
+              "decryption_unsupported": "Decryption is not supported for this backup. The backup will be encrypted downloaded and can't be opened. To restore it, you will need the encryption key.",
```

```suggestion
              "decryption_unsupported": "Decryption is not supported for this backup. The downloaded backup will remain encrypted and can't be opened. To restore it, you will need the encryption key.",
```
The original wording feels wrong. "downloaded encrypted" would be more correct, but is still extremely awkward phrasing. Hence this suggested phrasing.

---

Thanks @KevinCathcart!